package.json

```
@@ -1,76 +1,70 @@
-    "format": "prettier --check \"./**/*.{ts,tsx,js,css,json,md}\"",
-    "format:fix": "prettier --write  \"./**/*.{ts,tsx,js,css,json,md}\"",
-    "test": "jest --passWithNoTests",
-    "test:ci": "jest --passWithNoTests --ci",
-    "test:watch": "jest --passWithNoTests --watch",
-    "release": "cross-env HUSKY_SKIP_HOOKS=1 standard-version",
-    "release:next": "cross-env HUSKY_SKIP_HOOKS=1 standard-version --prerelease next"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/adobe/magento-storefront-event-collector"
-  },
-  "keywords": [],
-  "author": {
-    "name": "Brad Garropy",
-    "email": "bradgarropy@gmail.com",
-    "url": "https://bradgarropy.com"
-  },
-  "license": "SEE LICENSE IN LICENSE",
-  "bugs": {
-    "url": "https://github.com/adobe/magento-storefront-event-collector/issues"
-  },
-  "homepage": "https://github.com/adobe/magento-storefront-event-collector#readme",
-  "devDependencies": {
-    "@adobe/adobe-client-data-layer": "1.1.3",
-    "@types/jest": "26.0.15",
-    "@typescript-eslint/eslint-plugin": "^4.2.0",
-    "@typescript-eslint/parser": "^4.2.0",
-    "babel-loader": "^8.1.0",
-    "commitizen": "4.2.2",
-    "cross-env": "^7.0.3",
-    "cz-conventional-changelog": "3.3.0",
-    "eslint": "^7.9.0",
-    "html-webpack-plugin": "^5.3.1",
-    "husky": "4.3.0",
-    "jest": "26.6.1",
-    "prettier": "2.2.1",
-    "sass-loader": "^10.0.2",
-    "standard-version": "9.1.0",
-    "ts-jest": "26.4.3",
-    "ts-loader": "^8.0.4",
-    "typescript": "^4.0.3",
-    "webpack": "^5.26.0",
-    "webpack-cli": "^4.5.0",
-    "webpack-dev-server": "^4.0.0-beta.0"
-  },
-  "husky": {
-    "hooks": {
-      "prepare-commit-msg": "exec < /dev/tty && git cz --hook || true"
+    "name": "@adobe/magento-storefront-event-collector",
+    "version": "0.0.1",
+    "description": "Magento storefront event collector",
+    "main": "./dist/index.js",
+    "module": "./dist/index.js",
+    "types": "./dist/types/index.d.ts",
+    "files": [
+        "dist/"
+    ],
+    "scripts": {
+        "start": "webpack serve",
+        "build": "webpack --config webpack.config.js",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "format": "prettier --check \"./**/*.{ts,tsx,js,css,json,md}\"",
+        "format:fix": "prettier --write  \"./**/*.{ts,tsx,js,css,json,md}\"",
+        "test": "jest --passWithNoTests",
+        "test:ci": "jest --passWithNoTests --ci",
+        "test:watch": "jest --passWithNoTests --watch",
+        "release": "cross-env HUSKY_SKIP_HOOKS=1 standard-version",
+        "release:next": "cross-env HUSKY_SKIP_HOOKS=1 standard-version --prerelease next"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/adobe/magento-storefront-event-collector"
+    },
+    "keywords": [],
+    "author": {
+        "name": "Brad Garropy",
+        "email": "bradgarropy@gmail.com",
+        "url": "https://bradgarropy.com"
+    },
```

I know you're leading the efforts here, but since it's going to be open source, shouldn't we have the author be Adobe to make it seem more legit? 


```suggestion
    "author": {
        "name": "Adobe",
        "url": "https://www.adobe.com/"
    },
```